wordpress/README.md

```diff
@@ -0,0 +1,280 @@
+# Quick Start
+
+This repository includes the necessary files to package and deploy [Wordpress](https://www.wordpress.org/), a open source content management system, into a private cloud environment using [Gravity](http://gravitational.com/gravity/).  
+
+This manifest uses the Open Elastic Block Store ([OpenEBS](https://openebs.io/) built into Gravity 7.0.0+.
+
+# System Requirements
+
+
+* A x86_64 Linux machine or a VM for building a Cluster Image that is running one of the [supported Linux distributions](requirements.md#linux-distributions).
+* Docker version 17 or newer. Run `docker info` before continuing to make sure
+  you have Docker up and running. We recommend following instructions on [installing Docker CE from Docker.com](https://docs.docker.com/install/)
+* You must be a member of the `docker` group. Run `groups` command to make sure
+  `docker` group is listed. If not, you can add yourself to the "docker" group via `sudo usermod -aG docker $USER`
+* You must have `git` installed to clone the example application repo.
+* A _target cluster_ of Linux nodes. It can be just one machine but this example quickstart will use three. The nodes in a target cluster must have at least 2GB of RAM and 40GB of free disk space. They must **_not_** have Docker or any other container runtime installed on them.
```

Comment:
I'm not sure about any other container runtime. For example, `podman` would be fine since it does not require a daemon to be running. And we only explicitly validate the presence of `docker`

Reply:
Changed